batch7crud-roll7/src/main/webapp/static/css/reset.css

```diff
@@ -0,0 +1,17 @@
+*{
+	padding: 0px;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unit for this zero value [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Azero-units)